hudi-common/src/main/java/org/apache/hudi/common/model/HoodieAvroIndexedRecord.java

```diff
@@ -47,40 +48,62 @@
  * This only use by reader returning.
  */
 public class HoodieAvroIndexedRecord extends HoodieRecord<IndexedRecord> {
+  private static final long serialVersionUID = 1L;
+  private static final AvroRecordContext AVRO_RECORD_CONTEXT = new AvroRecordContext();
```

Comment:
is this variable serialzied altogeter with this record? should we move it to just `AvroRecordContext`.

Reply:
Right, makes sense. I added an instance there that is accessed with `getFieldAccessorInstance` as a way to try to keep the use case clear to callers